### DISCUSSION TITLE
False positive : how to unblock

### DISCUSSION THREAD

**Discussion by ernie95** [AdguardTeam/AdGuardHome#5855] [Q&A]
Hi,

I have adguardhome on a freebsd machine (version 0.107.22 for adguardhome).

The following site :
http://maitressege.eklablog.com
is for education.

Adguardhome blocks the site. But I put a specific rule:
@@||http://maitressege.eklablog.com^$important

If I test in adguardhome, it tells me:
"filtered by Parental" (traduction from French language)

If I enable or disable Adguardhome, same issue : the page is blocked by adguardhome (the pecific webpage of adguardhome)

How can I authorize this web site ?

Thanks

**Comment by ainar-g** [comment]
AdGuard Home is a DNS-level blocker, so you should not include the scheme (`http://`) or any other part of the URL:

```adblock
@@|maitressege.eklablog.com^$important
```

**Comment by ernie95** [comment]
Thanks for this evident rule. I know it now.
It solves the point
